packages/ui/src/blocks/comment-timeline/comment-timeline.tsx

```diff
@@ -53,11 +57,11 @@ export function CommentTimeline({
       </ul>
       <div className="flex flex-col items-end">
         <ul className={clsx(styles.commentList, `space-y-2`)}>
-          {comment.replies?.map((reply) => (
+          {comment?.replies?.map((reply) => (
             <CommentBranch key={reply.id}>
               <CommentCard
                 commentId={reply.id}
-                content={reply.content}
+                content={reply.content as RTEValue}
```

Comment:
Remove `as RTEValue`

Reply:
<img width="791" alt="image" src="https://user-images.githubusercontent.com/7880675/204277411-e08e9ce7-6e66-468c-b3c8-267ecafa96ab.png">
Still need it